e2e/scenario_test.go

```diff
@@ -995,16 +983,13 @@ func Scenario_genericHasRightContainerdVersion(t *testing.T, distroName string,
 	})
 }
 
-func Scenario_genericHasRightRunCVersion(t *testing.T, distroName string, distroRelease string, expectedVersion string, distro datamodel.Distro) {
+func Scenario_genericHasRightRunCVersion(t *testing.T, vhd *config.Image, expectedVersion string) {
 	RunScenario(t, &Scenario{
-		Description: "Tests that " + distroName + " " + distroRelease + " has the right runc version",
+		Description: fmt.Sprintf("Tests that %s %s %s %s has the right runc version", vhd.OS, vhd.Name, vhd.Version, vhd.Arch),
```

Comment:
nit (non-blocking): maybe we could implement `String()` on `config.Image` so we always get the same format/representation in strings?